### ISSUE TITLE
[SF_ACC][JAWS][ui5-table]: Row count is one more than the actual in JAWS VPC/forms mode for growing Table 

### ISSUE THREAD

**Issue by GongRichard** [SAP/ui5-webcomponents#5813]
### **Bug Description**
For growing Table, the row count is not correct in JAWS VPC/forms mode, the row count will be one more than the actual row count.
Please fix it in wc version 1.7.x or 1.8.0.

### **Expected Behavior**
Should announce correct row count in JAWS VPC/forms mode. 

### **Steps to Reproduce**
1. Go to 'https://codesandbox.io/s/summer-pine-gvvyk7?file=/src/App.js'
2. Open JAWS with VPC mode
3. use arrowing down key to access the growing table
4. you will hear "table with 3 columns and 6 rows", actually, the table has 5 rows.
5. check the DOM element, you will find there has one additional row definition in growing table. That's the root cause.

### **Isolated Example**
https://codesandbox.io/s/summer-pine-gvvyk7?file=/src/App.js

### **Context**
- UI5 Web Components version: {1.7.1}
- OS/Platform: {...}
- Browser: {...}
- Affected component: {ui5-table}

### **Log Output / Stack Trace / Screenshots**
![image](https://user-images.githubusercontent.com/13098801/190308068-a82e27ee-3b5a-4d50-afa1-db583c0b44c2.png)
<img width="1458" alt="image" src="https://user-images.githubusercontent.com/13098801/190308689-008cb699-2059-42c4-a789-20a9bac89152.png">

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: {SAP SuccessFactors}
- Business impact: {...}

**Comment by tsanislavgatev** [comment]
Hello colleagues,

Can you please check this issue?
It's also reproducible with Mac VoiceOver.

Best Regards,
Tsanislav

**Comment by niyap** [comment]
Hello Colleagues,

According to the current implementation, when the "growing" property is set to true, no matter whether the mode is Button or Scroll, we render an additional <tr> HTML element. If the mode is "Button", the "More" button is placed within it and if the mode is "Scroll" we render <tr tabindex="-1" class="ui5-table-end-row">.
That row is count in the total rows count and that is the reason for the one more row in the total count announced my JAWS.

We have to analyse whether it is possible, to do not render a <tr> when the ui5-table is growing and the "More" button to not be a part of the ui5-table. Enhancing the implementation in that way, we have to make the keyboard handling fully functional - navigating with the arrows between the rows, we have to be able to access the "More" button.

Could you please look over?

Kind Regards,
Niya

**Comment by GongRichard** [comment]
Hi @hristop ,
Any ETA of this issue? It's long time for this issue, we have some a11y issues depend on this issue.
Thanks,
Richard

**Comment by hristop** [comment]
Dear @GongRichard ,

We are writing to inform you about an important development regarding our ui5-table component. After careful consideration and received feedback, we have decided to focus on re-implementing and improving the existing table component based on the gathered feature requests and requirements.
Here are some details about this decision and how it may affect our ongoing development efforts.
 
The decision was made based on several factors, including scalability, performance, accessibility and the ability to meet future feature requirements. Our team has recognised the limitations of the current table component and after careful consideration, we concluded that it would be more effective to develop a new table component from scratch rather than invest more time in enhancing the current one.
 
While this decision allows us to provide a better user experience and meet evolving needs, I want to acknowledge that it may have some implications for the feature requests that have been made for the current ui5-table component. We understand that stakeholders have invested time and effort in suggesting improvements and new features. Rest assured that we value your input, and we will make every effort to take these feature requests into consideration during the development of the new ui5-table component.
 
However, it is important to note that the new table component may introduce backward incompatible changes. These changes may impact the way the current table component functions, including its API and visual representation. We understand that this may require adjustments and adaptations on your end. Our team will strive to provide documentation and guidance to facilitate a smooth transition to the new table component.
 
We are excited about the opportunities this transition will bring and the positive impact it will have on our product.
 
Should you have any questions or concerns regarding this transition or its implications, please don't hesitate to reach out.
 
Thank you for your understanding and continued support as we embark on this important transition.
Unfortunately for now we are not able to provide concrete timelines but we are currently working on a first preview version of the new WebComponent which should already offer a first impression of the concepts behind. We have gathered the feature requests and we surely have them in mind while developing the new component. 

Best Regards,
Hristo

**Comment by simlin** [comment]
Hi,

In the 2.0 version of the UI5 Web Components we introduced a new Table component:
https://sap.github.io/ui5-webcomponents/nightly/components/Table/

The table is still in an experimental phase and certain APIs might slightly change. Nevertheless we ask you to try out the new component already now. The earlier we get some feedback the better :)

We optimized the internal structure of the table, the usage of the ARIA role model and the keyboard support for a better accessibility.

Best Regards
Jens
